features/fields-fout-cases.feature

```diff
@@ -258,7 +265,7 @@ Rule: De Fields parameter bevat het pad naar de op te vragen velden
     Als personen wordt gezocht met de volgende parameters

```

Comment:
De titel van dit scenario zou n.m.m. moeten zijn:

`opgegeven pad is niet uniek en leidt naar meer dan één veld`

Reply:
Aangepast